set.mm

```diff
@@ -614237,6 +614271,250 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
+  rsubeulem1 $p |- ( 0 + ( 0 -R ( 0 + 0 ) ) ) = ( 0 -R 0 ) $=
+    ( cc0 crsub co caddc wceq 0cnALT2 cr wcel 0re readdcli rnegcl ax-mp addassi
+    recni rnegid eqtr3i wb rnegadd mp2an mpbir eqcomi ) AABCZAAAADCZBCZDCZUBUEE
+    ZAUEDCZAEZUCUDDCZUGAAAUDFFUDUCGHZUDGHAAIIJZUCKLZNMUJUIAEUKUCOLPAGHUEGHUFUHQ
+    IAUDIULJAUERSTUA $.
+
+  $( Lemma for ~ rsubeu .  A value which when added to ` A ` , results in
+     ` B ` .  (Contributed by Steven Nguyen, 7-Jan-2023.) $)
+  rsubeulem2 $p |- ( ( A e. RR /\ B e. RR ) ->
+    ( A + ( ( 0 -R A ) + ( ( 0 -R ( 0 + 0 ) ) + B ) ) ) = B ) $=
+    ( cr wcel wa cc0 crsub co caddc rnegid adantr oveq1d simpl recnd rnegcl 0re
+    wceq readdcli mp1i addassd simpr rsubeulem1 oveq1i 0red rneg0addid1 3eqtr3a
+    readdcld adantl 3eqtr3d ) ACDZBCDZEZAFAGHZIHZFFFIHZGHZBIHZIHFUQIHZAUMUQIHIH
+    BULUNFUQIUJUNFQUKAJKLULAUMUQULAUJUKMNULUMUJUMCDUKAOKNULUQULUPBUOCDUPCDULFFP
+    PRUOOSZUJUKUAZUGNTULFUPIHZBIHFFGHZBIHZURBVAVBBIUBUCULFUPBULFULUDNULUPUSNULB
+    UTNTUKVCBQUJBUEUHUFUI $.
+
+  ${
+    $d x A $.  $d x B $.
+    $( Existential uniqueness of real differences.  (Contributed by Steven
+       Nguyen, 7-Jan-2023.) $)
+    rsubeu $p |- ( ( A e. RR /\ B e. RR ) -> E! x e. RR ( A + x ) = B ) $=
+      ( cr wcel wa simpr simpl cc0 crsub caddc wceq wrex rnegcl adantr readdcli
+      co cv 0re readdcld mp1i rsubeulem2 oveq2 eqeq1d rspcev syl2anc rnegmod )
+      BDEZCDEZFZACBUHUIGZUHUIHUJIBJQZIIIKQZJQZCKQZKQZDEBUPKQZCLZBARZKQZCLZADMUJ
+      ULUOUHULDEUIBNOUJUNCUMDEUNDEUJIISSPUMNUAUKTTBCUBVAURAUPDUSUPLUTUQCUSUPBKU
+      CUDUEUFUG $.
+
+    $( Closure for real subtraction.  Based on ~ subcl .  (Contributed by NM,
+       10-May-1999.)  (Revised by Steven Nguyen, 7-Jan-2023.) $)
+    rsubcl $p |- ( ( A e. RR /\ B e. RR ) -> ( A -R B ) e. RR ) $=
+      ( vx cr wcel wa crsub co cv caddc wceq crio rsubval rsubeu ancoms riotacl
+      wreu syl eqeltrd ) ADEZBDEZFZABGHBCIJHAKZCDLZDCABMUBUCCDQZUDDEUATUECBANOU
+      CCDPRS $.
+  $}
+
+  ${
+    $d x A $.  $d x B $.  $d x C $.
+    $( Relation between real subtraction and addition.  Based on ~ subadd .
+       (Contributed by NM, 20-Jan-1997.)  (Revised by Steven Nguyen,
+       7-Jan-2023.) $)
+    rsubadd $p |- ( ( A e. RR /\ B e. RR /\ C e. RR ) ->
+                   ( ( A -R B ) = C <-> ( B + C ) = A ) ) $=
+      ( vx cr wcel w3a crsub co wceq cv caddc crio wb wa rsubval eqeq1d 3adant3
+      wreu rsubeu oveq2 riota2 sylan2 3impb 3com13 bitr4d ) AEFZBEFZCEFZGABHIZC
+      JZBDKZLIZAJZDEMZCJZBCLIZAJZUGUHUKUPNUIUGUHOUJUOCDABPQRUIUHUGURUPNZUIUHUGU
+      SUHUGOUIUNDESUSDBATUNURDECULCJUMUQAULCBLUAQUBUCUDUEUF $.
+  $}
+
+  ${
+    rsubaddd.1 $e |- ( ph -> A e. RR ) $.
+    rsubaddd.2 $e |- ( ph -> B e. RR ) $.
+    rsubaddd.3 $e |- ( ph -> C e. RR ) $.
+    $( Relationship between subtraction and addition.  (Contributed by Mario
+       Carneiro, 27-May-2016.)  (Revised by Steven Nguyen, 8-Jan-2023.) $)
+    rsubaddd $p |- ( ph -> ( ( A -R B ) = C <-> ( B + C ) = A ) ) $=
```

Comment:
"Based on ~ subaddd. " should be added, " (Contributed by Mario Carneiro, 27-May-2016.)  (Revised by Steven Nguyen, 8-Jan-2023.) " can be replaced by  (Contributed by Steven Nguyen, 8-Jan-2023.)

Reply:
OK now